README.md

```diff
@@ -1 +1,188 @@
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [Walk-Through Video](#walk-through-video)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Test](#test)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Ruby-Catalog-Project] <a name="about-project"></a>
+In this project, a console app is created that will help you to keep a record of different types of things you own: books, music albums, movies, and games. The data is stored in JSON files 
+and a database with tables structure analogical is also created.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[List all books]**
+- **[List all music albums]**
+- **[List of games]**
+- **[List all genres (e.g 'Comedy', 'Thriller')]**
+- **[List all labels (e.g. 'Gift', 'New')]**
+- **[List all authors (e.g. 'Stephen King')]**
+- **[Add a book]**
+- **[Add a music album]**
+- **[Add a game]**
+- **[All data preserved by saving collections in .json files.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Walk-Through Video -->
+
+## Walk-Through Video <a name="walk-through-video"></a>
+
+[Video](https://drive.google.com/file/d/1893S8dN7oHCV1H5Slgo72zPYZZ1CJS1M/view?usp=drive_link)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby Installed on System.
+- Terminal
+
+### Setup
+
+
+Clone this repository to your desired folder:
+```sh
+https://github.com/zunairkhan811/Ruby-Catalog-Project.git
+```
+cd my-folder
+```sh
+  cd Ruby-Catalog-Project
+```
+### Install
+Install the project using the following 
+```sh
+bundle install
+```
+### Usage 
+To run the project, execute the following command:
+```sh
+ruby main.rb
+```
+
+### Test
+To run tests, execute the following command:
+```sh
+  rspec
+```
+
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+
+- GitHub: [@zunairkhan811](https://github.com/zunairkhan811)
+- Twitter: [@zunairkhan811](https://twitter.com/zunairkhan811)
+- LinkedIn: [mzunairkhan](https://www.linkedin.com/in/mzunairkhan)
+
+👤 **Lawrence Amoafo**
+
+- GitHub: [@Kwaku28](https://github.com/Kwaku28)
+- Twitter: [@kwakuamoafo](https://twitter.com/kwakuamoafo)
+- LinkedIn: [Lawrence Amoafo](https://linkedin.com/in/lawrenceamoafo)
+
```

Comment:
- [ ] It would be great if you fix this LinkedIn link because right now it is broken 😨 KIndly fix it to make your readme descriptive and professional 👍

Reply:
Link updated